vendor/blockchain_contracts/src/rfc003/secret.rs

```diff
@@ -0,0 +1,171 @@
+use crate::rfc003::secret_hash::SecretHash;
```

Comment:
:x: I think this module should stay inside `comit_node`. The blockchain contracts only need a secret hash and that one can be a `String` or very simple new-type.

Reply:
The tests of such contracts still need to handle the secret and its hash. Because we wants the contracts to be tested live in this crate then I believe that full `SecretHash` struct needs to be copied across and cannot be replaced with a dummy simple `String` type.

Reply:
Note I think it's fine if a copy pasta of this is here but I note it should not be exported.
Actually this crate shouldn't export anything right? It's just the cli tool.

Reply:
@LLFourn hum, do you understand that `crate::` refers to `blockchain_contracts`? It's not imported from `comit_node`. In any case, it's now fixed.

Reply:
Meh, I had hoped you would get rid of all the unused code. It was created for very different purposes and IMO shouldn't be used here.

Reply:
Meh, I had hoped you would get rid of all the unused code. It was created for very different purposes and IMO shouldn't be used here.